sonic-slave/Dockerfile

```diff
@@ -213,7 +211,17 @@ RUN apt-get update && apt-get install -y \
         linuxdoc-tools \
         lynx \
         texlive-latex-extra \
-        texlive-latex-recommended
+        texlive-latex-recommended\
+# For python-click build
+        python-sphinx \
+        python-docutils \
+        python3-all \
+        python3-setuptools \
+        python3-sphinx \
+        python3-docutils \
+        python3-requests \
+        python3-pytest \
+        python3-colorama
```

Comment:
some are already installed.

Reply:
It doesn't hurt to specify them more than once, and this way it's explicit which packages are required by other packages. If in the future we have no need for a package and someone removes its dependencies here, shared dependencies will still be listed under the other packages which require them.